katas/content/teleportation/entangled_pair/index.md

```diff
@@ -0,0 +1,5 @@
+**Input:** 
+Two qubits qAlice and qBob, each in $|0\rangle$ state.
```

Comment:
I didn't parse these as variable names at a glance, so let's highlight them to make them stand out from the text a bit:
```suggestion
Two qubits `qAlice` and `qBob`, each in $|0\rangle$ state.
```

Reply:
Added